src/packages/client/src/__tests__/integration/happy/native-types-postgres/test.ts

```diff
@@ -25,7 +25,7 @@ test('native-types-postgres A: Integer, SmallInt, BigInt, Serial, SmallSerial, B
     name: 'Bob',
     int: 1,
     sInt: 3,
-    bInt: 12312312,
+    bInt: BigInt(12312312),
```

Comment:
Could be more interesting to push the safe limit to validate that it works as intended like: 

```
BigInt(Number.MAX_SAFE_INTEGER) * 2n;
// 9007199254740991 
// = 18014398509481982n

Reply:
:+1: will add that